File: Turret-Shot/Turret Shot/GameMechanics/Shot.swift
//
//  Shot.swift
//  Turret Shot
//
//  Created by 90307297 on 2/17/22.
//

import Foundation

class Shot:MovingObject{
    private var owner: Player
    

    init(o: Player){
        self.owner = o;
        
        super.init(xP: o.xPosition + o.xOrientation, yP: o.yPosition + o.yOrientation, xO: o.xOrientation, yO: o.yOrientation)
        
      }
}

File: Turret-Shot/Shot.java
public class Shot extends MovingObject{
  private Player owner;

  public Shot(Player o){
    super(o.getXPosition() + o.getXOrientation(), o.getYPosition() + o.getYOrientation(), o.getXOrientation(), o.getYOrientation());

    this.owner = o;
  }

}